BootloaderCommonPkg/Library/MmcAccessLib/MmcAccessLib.inf

```diff
@@ -38,8 +38,11 @@
   IoLib
   DebugLib
   TimerLib
+  IoMmuLib
 
 [Pcd]
   gPlatformCommonLibTokenSpaceGuid.PcdEmmcBlockDeviceLibId
   gPlatformCommonLibTokenSpaceGuid.PcdEmmcMaxRwBlockNumber
   gPlatformCommonLibTokenSpaceGuid.PcdEmmcHs400SupportEnabled
+  gPlatformCommonLibTokenSpaceGuid.PcdDmaBufferSize

```

Comment:
Better to update copyright year to all updated file

Reply:
pushed a new patch to fix it.